### DISCUSSION TITLE
feature: Mastodon comments plugin

### DISCUSSION THREAD

**Discussion by happybeing** [GetPublii/Publii#1986] [Ideas]
### Discussed in https://github.com/GetPublii/Publii/discussions/1857

<div type='discussions-op-text'>

<sup>Originally posted by **happybeing** December  7, 2024</sup>
I've just set up my Publii blog and the only thing that it needs now is a way for people to comment - that doesn't really on a commercial service. 

For me the best option would be Mastodon / ActivityPub. I believe this exists for WordPress so I'm hoping this would also be feasible for Publii.

I realise it would have to work differently so I imagine that when you create a post you would have a way for it to be posted to your Mastodon account and that Publii would store the toot URL with metadata for the post. 

A comment plugin could use the URL to let a commenter create a reply from their own Mastodon account, which would be displayed on the Publii blog as a comment (by fetching all the replies to the toot and showing them as comments). </div>

**Comment by s3nnet** [comment]
Yes, please
